### PR TITLE
ddl: Make reorg_test stable

### PR DESCRIPTION
**Pull request by zimulala**
Fix the problem as follows:
```
[Unit Test] reorg_test.go:74:
[Unit Test]     c.Assert(err, IsNil)
[Unit Test] ... value *terror.Error = &amp;terror.Error{class:2, code:7, message:"wait for reorganization timeout", args:[]interface {}(nil), file:"", line:0} ("[ddl:7]wait for reorganization timeout")
```

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: ddl/reorg_test.go [hunk 1]
```diff
@@ -70,6 +70,8 @@ func (s *testDDLSuite) TestReorg(c *C) {
 	c.Assert(err, NotNil)
 
 	<-done
+	// Make sure the function of f is returned.
+	time.Sleep(5 * time.Millisecond)
```
Comment (ngaut) [inline ddl/reorg_test.go]: Why we need to sleep, we already got singal from the "done" channel

Comment (zimulala) [inline ddl/reorg_test.go]: With the function really out there are still a little time. So it is only occasionally go wrong.

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**